### ISSUE TITLE
[ISSUE] Windows 2016 with docker will not install due to failed BITS transfer

### ISSUE THREAD

**Issue by billykwooten** [OneGet/MicrosoftDockerProvider#33]
### Replicate Issue
```
Install-Module -Name DockerMsftProvider -Force
Unregister-PackageSource -ProviderName DockerMsftProvider -Name DockerDefault -Erroraction Ignore
Register-PackageSource -ProviderName DockerMsftProvider -Name Docker -Location https://download.docker.com/components/engine/windows-server/index.json
Install-Package -Name docker -ProviderName DockerMsftProvider -Source Docker -Force
```

### Environment
Windows Server 2016
VMWare ESXi 6.5

### Issue Code Block
```
PS C:\Program Files\docker> Install-Package -Name docker -ProviderName DockerMsftProvider -Verbose
-Update
VERBOSE: Importing package provider 'DockerMsftProvider'.
VERBOSE: Using the provider 'DockerMsftProvider' for searching packages.
VERBOSE: Download size: 0.01MB
VERBOSE: Free space on the drive: 24860.9MB
VERBOSE: Downloading https://download.docker.com/components/engine/windows-server/index.json to
C:\Users\ADMINI~1\AppData\Local\Temp\1\DockerMsftProvider\Docker_DockerSearchIndex.json
VERBOSE: About to download
WARNING: Cannot find path
'C:\Users\ADMINI~1\AppData\Local\Temp\1\DockerMsftProvider\Docker_DockerSearchIndex.json' because
it does not exist.
WARNING: Save-HTTPItem: Bits Transfer failed. Job State: BG_JOB_STATE_TRANSIENT_ERROR ExitCode = 0
```

**Comment by jclagache** [comment]
Should be fixed whith #29

**Comment by enderb-ms** [comment]
BITS Transfer has now been removed from the provider. If you install DockerMsftProvider from the PowerShell Gallery, you should get version 1.0.0.3 which has this fix. You can also use `Update-Module -Name "DockerMsftProvider"` to update your provider.

**Comment by tomasquith** [comment]
@enderb-ms Sorry to jump in here, it seems I'm getting an issue related to this:

<img width="836" alt="screen shot 2018-02-22 at 10 33 47" src="https://user-images.githubusercontent.com/7528031/36533619-f8b3eaba-17bb-11e8-9462-473629974808.png">

Am I doing anything wrong given this provider fix?

**Comment by Skimo0412** [comment]
I'm seeing the same.  I installed version 1.0.0.3 yesterday.  nstall-Package : The specified module 'C:\Program Files\WindowsPowerShell\Modules\DockerMsftProvider\1.0.0.3\SaveHTTPItemUsingBITS.psm1' was not loaded because no valid module file was 
found in any module directory.
    + CategoryInfo          : ResourceUnavailable: (C:\Program File...mUsingBITS.psm1:String) [Install-Package], Exception
    + FullyQualifiedErrorId : Modules_ModuleNotFound,Microsoft.PowerShell.Commands.ImportModuleCommand,Microsoft.PowerShell.PackageManagement.Cmdlets.InstallPackage

**Comment by enderb-ms** [comment]
@tomasquith and @Skimo0412 thank you for reporting those. That error has been fixed with PR #38 and the fix has been pushed to the PS Gallery

**Comment by tomasquith** [comment]
Thanks @enderb-ms, I managed to prove this on Friday - cheers for the quick fix! :)
